js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+function abbreviateString(str) {
```

Comment:
and please change the name from `str` to something more meaningful.
Like atleast `string` 😀

Reply:
@varora1406 How about something more meaningful like `fullName`  or `sentence` of that's what we always receive. Though, in this case, `fullName` will not suit as we can pass a sentence.

Reply:
yes `fullname` won't suit best, as this is not limited to names.

it can be said as `phrase`. After all we need abbreviation only where there are group of words.